.github/workflows/labeler.yml

```diff
@@ -0,0 +1,72 @@
+# .github/labeler.yml
+
+# Label PRs that modify documentation files
+Documentation:
+  - changed-files:
```

Comment:
Consider adding comments to clarify the purpose and usage of each label configuration for future maintainability.